EvidenceApi.Tests/V1/E2ETests/DocumentSubmissionsTest.cs

```diff
@@ -85,7 +86,7 @@ public async Task CanUpdateDocumentSubmissionStateWithValidParameters()
             var result = JsonConvert.DeserializeObject<DocumentSubmissionResponse>(json);
 
             var documentType = TestDataHelper.DocumentType("passport-scan");
-            var staffSelectedDocumentType = TestDataHelper.DocumentType("drivers-licence");
+            var staffSelectedDocumentType = TestDataHelper.GetStaffSelectedDocumentTypeByTeamName("drivers-licence", teamName);
```

Comment:
See above -- this test was failing because it was grabbing the first `drivers-licence` it found, instead of the team's